src/cloudant/_common_util.py

```diff
@@ -268,16 +268,24 @@ def get_docs(r_session, url, encoder=None, headers=None, **params):
 #pylint: disable=unused-argument
 def append_response_error_content(response, **kwargs):
     """
-    Provides a helper to add HTTP response error and reason messages.
+    Provides a helper to act as callback function for the response event hook
+    and add a HTTP response error with reason message to `response.reason`.
+    The `response` and `**kwargs` are necessary for this function to
```

Comment:
I think you meant to use double ticks rather than single here (response.reason, response, **kwargs).  In Sphinx using single ticks will _italisize_ while double ticks will `highlight`.  If you meant to _italisize_ then ignore this comment.